src/components/confirmCredentialsForm/confirmCredentialsFormHook.tsx

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+import { useTranslation } from 'react-i18next';
+
+export const useConfirmCredentialsSchema = () => {
+  const { t } = useTranslation();
+  const confirmSchema = yup
```

Comment:
empty line above

Reply:
fixed in [5309628](https://github.com/ZenBit-Tech/faq_team_fe/pull/8/commits/53096286bb449eb7af490481ea7d5d267d756c46)